src/test/java/com/epam/tc/hw4/steps/ActionStep.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw4.steps;
+
+import com.epam.tc.hw4.AbstractChromeTest;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ActionStep extends AbstractChromeTest {
+    public ActionStep(WebDriver webDriver) {
+        super();
+    }
+
+    @Step("Open test site by URL")
+    public void openWebSite(String website) {
+        getWebDriver().get(website);
+    }
+
+    @Step("Perform login as {username}|{password}")
+    public void performLogin(String user, String password) {
+        //mainPage = new MainPage(webDriver);
+        mainPage.login(user, password);
+    }
+
+    @Step("Switch to the {frame}")
+    public void switchToIframe(String iframe) {
+        getWebDriver().switchTo().frame(iframe);
```

Comment:
As I said in a previous review, you should create a method in your page object. Then you should call this method in your step method...